Projects not loading when using standalone config with empty workspace starter

Whenever I move a project to it's standalone config move, the extension stops listing projects, running their command and various things.

1. Right after you move a project to it's standalone config or create a new one, the project stops listing all projects. 
`The "path" argument must be of type string. Received undefined`
2. Clicking Run-many/affected results in this error - 
`None of your workspace projects have an architect command`

>  NX  Report complete - copy this into the issue template

  Node : 16.4.0
  OS   : linux x64
  pnpm : 6.9.1
  
  nx : Not Found
  @nrwl/angular : Not Found
  @nrwl/cli : 12.5.7
  @nrwl/cypress : 12.5.7
  @nrwl/devkit : 12.5.7
  @nrwl/eslint-plugin-nx : 12.5.7
  @nrwl/express : Not Found
  @nrwl/jest : 12.5.7
  @nrwl/linter : 12.5.7
  @nrwl/nest : Not Found
  @nrwl/next : 12.5.7
  @nrwl/node : 12.5.7
  @nrwl/react : 12.5.7
  @nrwl/schematics : Not Found
  @nrwl/tao : 12.5.7
  @nrwl/web : 12.5.7
  @nrwl/workspace : 12.5.7
  @nrwl/storybook : 12.5.7
  @nrwl/gatsby : Not Found
  typescript : 4.2.4

Hmm... Nx is [requiring '@angular-devkit/schematics'](https://github.com/nrwl/nx/blob/master/packages/workspace/src/utils/project-type.ts#L1) to read the standalone configuration workspace.

```
Error: Cannot find module '@angular-devkit/schematics'
Require stack:
- /Users/sandikbarr/code/nrwl/test/split-config/node_modules/@nrwl/workspace/src/utils/ast-utils.js
- /Users/sandikbarr/code/nrwl/test/split-config/node_modules/@nrwl/workspace/src/utils/project-type.js
- /Users/sandikbarr/code/nrwl/test/split-config/node_modules/@nrwl/workspace/index.js
- /Users/sandikbarr/code/nrwl/nx-console/dist/apps/vscode/main.js
```

Dang. Using the standalone config shouldn't have a dependency on angular-devkit. We'll look at this further and get it addressed. Appreciate the report, thanks.

--- a/libs/vscode/nx-workspace/src/lib/get-nx-workspace-package.ts
+++ b/libs/vscode/nx-workspace/src/lib/get-nx-workspace-package.ts
@@ -4,13 +4,21 @@ import type * as NxWorkspacePackage from '@nrwl/workspace';
 /**
  * Get the local installed version of @nrwl/workspace
  */
-export function getNxWorkspacePackage(): typeof NxWorkspacePackage {
+export function getNxWorkspacePackageFileUtils(): typeof NxWorkspacePackage {
   const workspacePath = dirname(
     WorkspaceConfigurationStore.instance.get('nxWorkspaceJsonPath', '')
   );
 
   // webpack hacks..
   return eval('require')(
-    join(workspacePath, 'node_modules', '@nrwl', 'workspace')
+    join(
+      workspacePath,
+      'node_modules',
+      '@nrwl',
+      'workspace',
+      'src',
+      'core',
+      'file-utils'
+    )
   );
 }

--- a/libs/vscode/nx-workspace/src/lib/verify-workspace.ts
+++ b/libs/vscode/nx-workspace/src/lib/verify-workspace.ts
@@ -9,7 +9,7 @@ import {
 import { window } from 'vscode';
 import { dirname, join } from 'path';
 import { WorkspaceConfigurationStore } from '@nx-console/vscode/configuration';
-import { getNxWorkspacePackage } from './get-nx-workspace-package';
+import { getNxWorkspacePackageFileUtils } from './get-nx-workspace-package';
 
 export function verifyWorkspace(): {
   validWorkspaceJson: boolean;
@@ -83,7 +83,7 @@ function readNxWorkspaceConfig(basedir: string, workspaceJsonPath: string) {
   try {
     const cachedWorkspaceJson = cacheJson(workspaceJsonPath).json;
     if (!cachedWorkspaceJson) {
-      const workspace = getNxWorkspacePackage().readWorkspaceConfig({
+      const workspace = getNxWorkspacePackageFileUtils().readWorkspaceConfig({
         format: 'nx',
         path: basedir,
       } as any);

